Backend/app/rider-platform/rider-app/Main/src/Tools/Notifications.hs

```diff
@@ -1065,7 +1070,7 @@ getAllOtherRelatedPartyPersons :: ServiceFlow m r => SRB.Booking -> m [Person]
 getAllOtherRelatedPartyPersons booking = do
   case booking.tripCategory of
     Just (Trip.Delivery _) -> do
-      allBookingParties <- QBPL.findAllActiveByBookingId booking.id
+      allBookingParties <- QBPL.findAllByBookingId booking.id
```

Comment:
Why this change ?

Reply:
due to end ride case .. 
End ride fcm was not coming as all became inactive

Reply:
We should make them inActive after triggering all FCM and everything